<launch>
  <arg name="aruco_dictionary" default="DICT_4X4_100" doc="Dictionary to create. Check the readme to find a list of the available dictionary"/>
  <arg name="aruco_ids" doc="Comma separated string of aruco ids to generate. If -1 all dictionary code will be generated "/>

  <!-- Launch ROS Package Template Node -->
  <node pkg="aruco_description" type="generate_aruco_mesh.py" name="$(anon generate_aruco_mesh)" output="screen">
    <!-- <rosparam command="load" file="$(arg param_file)" /> -->
    <param name="aruco_dictionary" value="$(arg aruco_dictionary)" />
    <param name="aruco_ids" value="$(arg aruco_ids)" />
  </node>

</launch>
